lumigator/python/mzai/backend/lm_buddy/__init__.py

```diff
@@ -0,0 +1,3 @@
+from lumigator.python.mzai.backend.lm_buddy.buddy import LMBuddy
```

Comment:
I think we should move this out of the `backend` folder and make it it's own module so we can have independent deps between the backend API and lm-buddy jobs.

Reply:
That works for me, initially I debated between having it at `lumigator.python.mzai.backend` and `lumigator.python.mzai` like we did for the summarizer and felt that where the summarizer was, was in a very weird place in relation to the code, but perhaps it makes more sense logically since lm-buddy is larger and "further" away functionality-wise

Reply:
Another question is whether we see this retaining the lm-buddy name. I'm happy to keep it but feel like it might be less confusing to be named evaluator or something more in line with the names of lumigator components.

Reply:
I'm also good to re-name

Reply:
If we want to use it as just an evaluator, it makes sense to rename it that way and avoid confusion by removing non-eval entrypoints (namely finetuning).

Reply:
Will rename in next PR + remove finetuning